specifications/api/swagger.yaml

```diff
@@ -1193,6 +1211,19 @@ components:
         pledge: *stakePoolPledge
         metadata: *stakePoolMetadata
         retirement: *stakePoolRetirement
+        delisted:
+          type: boolean
```

Comment:
This is just a thought, but I would be inclined to use an enumeration type here:

- `delisted`
- `not_delisted`

Reply:
As explained above, I think that's excessive for API and json.